Source/JavaScriptCore/yarr/YarrPattern.cpp

```diff
@@ -1803,26 +1827,21 @@ class YarrPatternConstructor {
         }
     }
 
-    void setupDuplicateNamedCaptures()
+    void setupNamedCaptures()
     {
-        if (!m_pattern.m_numNamedSubpatterns)
+        if (!m_pattern.m_hasNamedCaptureGroups)
             return;
 
         // Finish padding out m_captureGroupNames vector.
         while (m_pattern.m_captureGroupNames.size() <= m_pattern.m_numSubpatterns)
             m_pattern.m_captureGroupNames.append(String());
 
-        m_pattern.m_numDuplicateNamedCaptureGroups = 0;
-
         for (auto& namedGroupIndecies : m_pattern.m_namedGroupToParenIndeces.values()) {
```

Comment:
nit: Looks like we've got a couple of different misspellings for `indices` in this code—might be nice to clean 'em up right away.

Reply:
I took care of the mispellings.